src/main.rs

```diff
@@ -85,7 +86,15 @@ fn settings<'a>() -> (ArgMatches<'a>, Vec<PathBuf>, String, String) {
         )
         .subcommand(SubCommand::with_name(LIST_TAGS_SUBCOMMAND).about("Lists all the tags"))
         .subcommand(
-            SubCommand::with_name(LIST_UNUSED_SUBCOMMAND).about("Lists the unreferenced tags"),
+            SubCommand::with_name(LIST_UNUSED_SUBCOMMAND)
+                .about("Lists the unreferenced tags")
+                .arg(
+                    Arg::with_name(LIST_UNUSED_ERROR_OPTION)
+                        .long(LIST_UNUSED_ERROR_OPTION)
+                        .short("e")
+                        .takes_value(false)
+                        .help("Exits with an error status code if any tags are unreferenced"),
```

Comment:
Can we change it to the following?

```rust
Arg::with_name(LIST_UNUSED_ERROR_OPTION)
    .short("e")
    .long(LIST_UNUSED_ERROR_OPTION)
    .help("Exits with an error status code if any tags are unreferenced"),
```

(Reordered long/short for consistency and removed `.takes_value(false)` which I believe is already the default.)